src/validators/duration_distance.rs

```diff
@@ -126,7 +126,15 @@ fn validate_speeds(gtfs: &gtfs_structures::Gtfs) -> Result<Vec<Issue>, gtfs_stru
                             .add_related_object(&*arrival.stop)
                             .details(&details)
                     });
-                    issue.push_related_object(trip);
+                    // crappy implementation of "only add route if not already there"
+                    // if the idea is good, we'll likely switch to some hash map
```

Comment:
I will move the function to where `push_related_object` is defined instead later, e.g. `push_unique_related_route`.

Reply:
exactly - I've updated the note to make this clearer and also mention what we were doing before (referring to trips), in case we want to easily change that back later for some reason.